src/styles/index.css

```diff
@@ -0,0 +1,95 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif, Arial;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card_image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.description {
+  font-weight: 500;
+  margin-bottom: 4px;
+  font-size: 12px;
+}
+
+.card_image--imac {
+  background-image: url(../images/imac.jpeg);
+}
+
+.product_code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.star_review {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.reviews {
+  display: flex;
+  white-space: nowrap;
+  font-size: 10px;
+  align-items: center;
+}
+
+.stars_active {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.text_price {
+  color: #616070;
+}
+
+.numeric_price {
```

Comment:
Class names should use hyphens instead of underscores according to the BEM naming convention. Consider renaming '.numeric_price' to '.numeric-price'.